phpunit.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://phpunit.de/manual/current/en/appendixes.configuration.html -->
+<phpunit
+	backupGlobals="false"
+	backupStaticAttributes="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+	processIsolation="false"
+	stopOnFailure="false"
+	syntaxCheck="false"
+	bootstrap="vendor/autoload.php">
+
+	<testsuites>
+		<testsuite name="Project Test Suite">
+			<directory>tests</directory>
```

Comment:
Maybe add `suffix="Test.php"` to the directory element? That should prevent TestCase.php from being included in the suite.